tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/files/yaml_example.py

```diff
@@ -0,0 +1,207 @@
+import os
+
+from ruamel import yaml
+
+import great_expectations as ge
+from great_expectations.core.batch import BatchRequest
+
+context = ge.get_context()
+data_dir_path = os.path.join("..", "data")
```

Comment:
Should this be `os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', 'data'))`?

Relative pathing can be weird sometimes so I'd rather be explicit and use abspath if possible.

Reply:
@cdkini I am following the pattern of the integration tests, which are enabled by `tests/integration/test_script_runner.py` -- there are provisions to set up the `data` directory in the way that it is referenced.  If we are to change this, then I would propose it to be a separate effort.  Thanks! /cc @Shinnnyshinshin